setup.py

```diff
@@ -49,6 +49,9 @@
 ONNX_NAMESPACE = os.getenv("ONNX_NAMESPACE", "onnx")
 ONNX_BUILD_TESTS = bool(os.getenv("ONNX_BUILD_TESTS") == "1")
 ONNX_DISABLE_EXCEPTIONS = bool(os.getenv("ONNX_DISABLE_EXCEPTIONS") == "1")
+ONNX_DISABLE_STATIC_REGISTRATION = bool(
```

Comment:
Is there a place where these options are documented?

Reply:
I am not aware of any, although these options are identical as the options in CMakeLists.txt: https://github.com/onnx/onnx/blob/391f5703abe1a0d5c8cbb0b38df079b0bacec079/CMakeLists.txt#L25-L28

which has a simple sentence as description.